db/migrate/20250509105746_create_department_skill_snapshots.rb

```diff
@@ -0,0 +1,10 @@
+class CreateDepartmentSkillSnapshots < ActiveRecord::Migration[8.0]
+  def change
+    create_table :department_skill_snapshots do |t|
```

Comment:
So I guess your plan is to go off the `created_at` for sorting the snapshots etc.

I was thinking why not add a `month` and `year` variable to the snapshots that more specifically mark them to be from a certain time... or maybe not.

We also can just use the actual `created_at` all the time even when displaying the snapshots... I am just thinking about what happens if the snapshot doesn't work for a while.... I don't know... useless rant over.